orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -222,11 +220,13 @@ def _get_session_config(self):
         # default to PRIORITY_BASED execution order
         session_options.execution_order = onnxruntime.ExecutionOrder.PRIORITY_BASED
         # 0:Verbose, 1:Info, 2:Warning. 3:Error, 4:Fatal. Default is 2.
-        session_options.log_severity_level = int(self._loglevel)
+        session_options.log_severity_level = int(self._debug_options.logging.log_level)
 
         # enable dumping optimized training graph
-        if self._save_onnx:
-            session_options.optimized_model_filepath = self._save_onnx_prefix + '_training_optimized.onnx'
+        if self._debug_options.save_onnx_models.save:
+            session_options.optimized_model_filepath = \
+                os.path.join(self._debug_options.save_onnx_models.directory,
```

Comment:
```suggestion
                os.path.join(self._debug_options.save_onnx_models.path,
```
Python chose "path" name over directory. Better to keep the terminology uniform